tests/testdata/provider/testdata_pg.sh

```diff
@@ -20,7 +20,7 @@ dropdb --if-exists $DB
 createdb $DB -E UTF8 -T template0 || exit 1
 for f in ${SCRIPTS}; do
   echo "Restoring $f"
-  psql -q --echo-errors -c "SET client_min_messages TO WARNING;" -f $f $DB -v ON_ERROR_STOP=1 || exit 1
+  psql -q -tA --echo-errors -c "SET client_min_messages TO debug;" -f $f $DB -v ON_ERROR_STOP=1 || exit 1
```

Comment:
seems "debug" is not a valid value: https://www.postgresql.org/docs/12/runtime-config-client.html#GUC-CLIENT-MIN-MESSAGES

Reply:
Thanks.
All in!